src/python/turicreate/toolkits/_internal_utils.py

```diff
@@ -90,6 +90,7 @@ def _find_only_column_of_type(sframe, target_type, type_name, col_name):
     raised. The name and type of the target column should be provided as
     strings for the purpose of error feedback.
     """
+    _raise_error_if_not_sframe(sframe,"dataset")
```

Comment:
If the `sframe` variable is not an SFrame, this will raise an error message claiming the name of the input parameter was "dataset". I don't think we want to assume the input parameter will always be name "dataset".

Sadly, I think we'll need to add this check to each `.create` method that doesn't already have it.

Reply:
Just fixed all files which didn't check the type error.